spec/lib/task-spec.js

```diff
@@ -326,6 +343,102 @@ describe("Task", function () {
             expect(task.serialize().job).to.deep.equal(task.job.serialize());
         });
     });
+    describe("sku rendering", function() {
```

Comment:
Still need some negative  test cases, for example:
(1) If env is avaiable, but lack the sku data, the rendering of sku should be failed.
(2) If both `env` and `sku` data is not available, the rendering for sku & env should be failed.
(3) If the node doesn't have the sku, the sku rendering should be fail but env rendering will success.
(4) Maybe more...